db/deploy/removed-panels.sql

```diff
@@ -0,0 +1,9 @@
+-- Deploy rams-core:removed-panels to pg
```

Comment:
As I did with the ``hotel`` plugin, I just removed the tables here, so that they won't be part of the core plugin.  They'll get automatically added if someone includes the ``panels`` repo, and we can include separate migrations there.